modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -15,7 +15,26 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. In the *Builder* section, after the URL is validated, an appropriate builder image is detected, indicated by a star, and automatically selected. For the `https://github.com/sclorg/nodejs-ex` Git URL, the Node.js builder image is selected by default. If a builder image is not auto-detected, select a builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
+. *Optional*: You can import a Devfile or a Dockerfile through your Git repository to further customize your deployment.
```

Comment:
```suggestion
. Optional: You can import a Devfile or a Dockerfile through your Git repository to further customize your deployment.
```
Please search the openshift-docs repo for "Optional" to see the precedents for how it is used.

Reply:
Fixed now.